qcodes/instrument_drivers/Harvard/Decadac.py

```diff
@@ -141,7 +188,7 @@ def _setvoltage(self, voltage, channel):
             mssg += ''.join(script) + runcmd
             self.visa_handle.write(mssg)
             sleep(0.0015*self.ramp_time)  # Required sleep.
```

Comment:
ok reload fixes this but I'm kind of surprised it can even make that error message :(